adapters/bevy/client/src/systems.rs

```diff
@@ -170,3 +170,11 @@ pub fn should_tick(resource: Res<ClientResource>) -> ShouldRun {
 pub fn finish_tick(mut resource: ResMut<ClientResource>) {
     resource.ticker.reset();
 }
+
+pub fn should_do_io<P: ProtocolType>(client: Res<Client<P, Entity>>) -> ShouldRun {
+    if client.connection_requested() {
```

Comment:
client.connection_requested() was removed in one of your earlier commits, commit e0fa83c7198622b3c0f401d442823b0b6061d2f7